crates/ink/codegen/src/generator/dispatch.rs

```diff
@@ -79,12 +79,7 @@ impl GenerateCode for Dispatch<'_> {
             #contract_dispatchable_messages_infos
             #constructor_decoder_type
             #message_decoder_type
-
-            #[cfg(not(test))]
```

Comment:
why remove this?

Reply:
because I've added that part directly in entry points